plugin/plugins/mijia/__init__.py

```
@@ -80,14 +75,73 @@ async def on_startup(self, **_):
                 self.logger.warning("注册静态UI失败，请检查 static/index.html 是否存在")
 
         return Ok({"status": "ready"})
-    
+
+    async def _background_load_credential(self):
+        """后台静默加载凭据，不阻塞插件启动"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)